docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -1120,6 +1121,21 @@ Returns UUID v4 string value which is unique per row.
 uuid()
 ```
 
+### `overlay`
+
+Returns the string which is replaced by another string from the specified position and specified count length
+
+```
+overlay(str PLACING substr FROM pos [FOR count])
+```
+
```

Comment:
Having the example made it much easier for me to see what was going on in postgres. Can we add an example here as well?

```suggestion
For example, `overlay('Txxxxas' placing 'hom' from 2 for 4) → Thomas`
```

Reply:
To resolve the conflict, I have added this, thanks